ee/clickhouse/queries/funnels/test/test_funnel_correlation.py

```diff
@@ -348,7 +348,7 @@ def test_correlation_with_properties_raises_validation_error(self):
         with self.assertRaises(ValidationError):
             FunnelCorrelation(filter, self.team)._run()
 
-    @test_with_materialized_columns(event_properties=[], person_properties=["$browser"])
+    @test_with_materialized_columns(event_properties=[], person_properties=["$browser", "$nice"])
```

Comment:
Sup? Mind adding the snapshot headers? Would make it clearer what's going on

Reply:
Ah, reverted this. The issue here is: I added `WITH` to capture select queries, which now catches all of these as well^. They are supposed to have JSONExtract in there, so just adding `verify_json_extract=False` to them.

Reply:
This is the crux: https://github.com/PostHog/posthog/pull/7008/files#diff-6b8ba77426159fc235e60dda5bd56c76ecf3ef5ec9b12a1376982fc55d7b198cR58

Reply:
thanks! :)